### ISSUE TITLE
Packages for Windows and Linux

### ISSUE THREAD

**Issue by wizardishungry** [WIZARDISHUNGRY/vcvrack-rtlsdr#22]
I could use a maintainer for Windows and Linux packages. Apply within!

Linux: blocked by #26 
Windows: @dizzisound built recent rev

**Comment by phdsg** [comment]
tried to build current master on my debian. fails with this output:
```
Makefile:29: warning: overriding recipe for target 'dist'
../../plugin.mk:36: warning: ignoring old recipe for target 'dist'
Package libusb-1.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libusb-1.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libusb-1.0' found
Package librtlsdr was not found in the pkg-config search path.
Perhaps you should add the directory containing `librtlsdr.pc'
to the PKG_CONFIG_PATH environment variable
No package 'librtlsdr' found
g++  -fPIC -I../../include -I../../dep/include -MMD -g -O3 -march=nocona -ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_LIN  -Wsuggest-override -std=c++11 -c -o build/src/sdr/SDR.cpp.o src/sdr/SDR.cpp
In file included from src/sdr/SDR.cpp:2:0:
src/sdr/rtl-sdr.h:3:21: fatal error: rtl-sdr.h: No such file or directory
 #include <rtl-sdr.h>
                     ^
compilation terminated.
../../compile.mk:53: recipe for target 'build/src/sdr/SDR.cpp.o' failed
make: *** [build/src/sdr/SDR.cpp.o] Error 1
```

**Comment by phdsg** [comment]
ok, installed the deps via apt. now i get this:
```
Makefile:29: warning: overriding recipe for target 'dist'
../../plugin.mk:36: warning: ignoring old recipe for target 'dist'
g++ -I/usr/include/libusb-1.0 -I/usr/include/ -fPIC -I../../include -I../../dep/include -MMD -g -O3 -march=nocona -ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_LIN  -Wsuggest-override -std=c++11 -c -o build/src/sdr/SDR.cpp.o src/sdr/SDR.cpp
cc -I/usr/include/libusb-1.0 -I/usr/include/ -fPIC -I../../include -I../../dep/include -MMD -g -O3 -march=nocona -ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_LIN  -c -o build/src/sdr/rtl-sdr.c.o src/sdr/rtl-sdr.c
src/sdr/rtl-sdr.c: In function ‘polar_disc_lut’:
src/sdr/rtl-sdr.c:648:31: warning: left shift of negative value [-Wshift-negative-value]
   return (cj > 0) ? 1<<13 : -1<<13;
                               ^~
src/sdr/rtl-sdr.c: In function ‘mark_shared_buffer’:
src/sdr/rtl-sdr.c:959:9: warning: statement with no effect [-Wunused-value]
  ss_busy[0];
  ~~~~~~~^~~
src/sdr/rtl-sdr.c: In function ‘RtlSdr_init’:
src/sdr/rtl-sdr.c:1858:21: warning: assignment from incompatible pointer type [-Wincompatible-pointer-types]
  radio->rack_buffer = &demod.rack_buffer;
                     ^
src/sdr/rtl-sdr.c:1804:8: warning: unused variable ‘freq’ [-Wunused-variable]
  char* freq = "99.5M";
        ^~~~
cc -I/usr/include/libusb-1.0 -I/usr/include/ -fPIC -I../../include -I../../dep/include -MMD -g -O3 -march=nocona -ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_LIN  -c -o build/src/sdr/convenience.c.o src/sdr/convenience.c
g++ -o plugin.so build/src/PQ.cpp.o build/src/sdr/SDR.cpp.o build/src/sdr/rtl-sdr.c.o build/src/sdr/convenience.c.o /usr/lib/x86_64-linux-gnu/libusb-1.0.a /usr/lib/x86_64-linux-gnu/librtlsdr.a -shared
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/librtlsdr.a(librtlsdr.c.o): relocation R_X86_64_PC32 against symbol `stderr@@GLIBC_2.2.5' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: Bad value
collect2: error: ld returned 1 exit status
../../compile.mk:43: recipe for target 'plugin.so' failed
make: *** [plugin.so] Error 1

```

**Comment by WIZARDISHUNGRY** [comment]
@phdsg try adding `-fPIC` to LDFLAGS in the Makefile? (I really should set up a vm for Linux testing.)

**Comment by phdsg** [comment]
same error...  
also, looking at the previous log, it seems -fPIC was already set.  
or does the error mean i have to compile the deps with -fPIC?

**Comment by wizardishungry** [comment]
@phdsg I'm moving the compilation issues over to a new issue #26.
